src/pydrex/io.py

```diff
@@ -747,7 +747,7 @@ def _parse_scsv_cell(func, data, missingstr=None, fillval=None):
         if fillval == "NaN":
             return func(np.nan)
         return func(fillval)
-    elif func == bool:
+    elif isinstance(func, bool):
```

Comment:
In this case, I wanted to literally check that `func` is the `bool()` builtin function. The `isinstance` check is changing this to check the type of `func`. I think a way to fix it and keep pre-commit happy is to check `func.__qualname__`.